### PR TITLE
alias-pof/121 - Alias Updates

### LINKED ISSUE #117: Add .ea alias
.ea for "expect approach" --> "expect $1 Runway $2 approach"

### LINKED ISSUE #133: Route finder alias
Fixed dep
```.routem .openurl https://flightaware.com/analysis/route.rvt?origin=KMIA&destination=$uc($1)```
```.routef .openurl https://flightaware.com/analysis/route.rvt?origin=KFLL&destination=$uc($1)```
```.routet .openurl https://flightaware.com/analysis/route.rvt?origin=KTPA&destination=$uc($1)```
```.routep .openurl https://flightaware.com/analysis/route.rvt?origin=KPBI&destination=$uc($1)```
```.routee .openurl https://flightaware.com/analysis/route.rvt?origin=KEYW&destination=$uc($1)```
```.router.openurl https://flightaware.com/analysis/route.rvt?origin=KRSW&destination=$uc($1)```

Custom dep
```.route .openurl https://flightaware.com/analysis/route.rvt?origin=$uc($1)&destination=$uc($2)```

Opens IFR route analyzer from flightaware to find quick routes for the destination.

### LINKED ISSUE #134: Pushback at your discretion PDC option
PDCs do not have an option to signal whether or not pushback will be at the pilots discretion, rather, it assumes it will be and states so. This can bring negative effects when a plane is parked at a gate that pushes onto a taxiway.

### LINKED ISSUE #135: Rename PDR aliases by airport, not by VDF
PDR aliases should be usable without specifying the departure fix, and just the airport. i.e. instead of `.HEDLYQ87`, you could do `.MIAQ87 / .MQ87` or `.FLLQ87 / .FQ87`. When they file the wrong departure fix, it's easier to find the right routing if you don't have to remember which fix should be used to get there.

### LINKED ISSUE #157: FLL PDCs have 3000 instead of 4000
Update FLL PDCs (such as `.pdcfaf` etc) to show the new initial altitude of 4k instead of the old 3k.

### PR DESCRIPTION
**Pull request by migte**
# Alias File Updates

## PDAR PDC 
Resolves #135.
Resolves #157.
Begins work on #121.

Shortened PDAR PDC aliases to work based off airfield rather than departure fix.

Before:
```
.wincoy280
.pdcwincoy280
```
After:
```
.my280
.pdcmy280
```
_"m" standing for Miami_

#### Resolves #133 
Alias to pull up the FlightAware IFR Route Analyzer

Major Field Example:
```
.routem EGLL
```
_Returns IFR routes from Miami International to London Heathrow_

Other Field Example:
```
.route KHST EGLL
```
_Returns IFR routes from North Perry to London Heathrow_

#### Resolves #117 
Expect approach alias.

Expect Approach Normal:
```
.hiea ILS 8R
```
_expect ILS Runway 8R approach._

Expect Approach w/Altimeter:
```
.hiaea KMIA ILS 8R
```
_Miami altimeter 29.92, expect ILS Runway 8R approach._

#### Resolves #134
All PDCs can now specify whether the pilot can pushback at their discretion or they must call the ground controller to push onto an active taxiway.

Current: 
```
.PDCAF MV G1
```
_...CTC 121.800 FOR TAXI..._

At Pilots Discretion:
```
.PDCAFPD MV G1
```
_...CTC 121.800 FOR TAXI, PUSHBACK AT PILOT'S DISCRETION..._

Call For Push:
```
.PDCAFCFP MV G1
```
_...CTC 121.800 FOR PUSHBACK..._

##### This works on all PDC variations

### CODE REVIEW

File: docs/alias/Alias.txt [hunk 1]
```diff
@@ -651,20 +731,20 @@ VRC/VSTARS/VERAM CONFIGURATION ALIASES
...
-.THNDRY280 .MSG ZMA_INFO THNDR JAYMC Y280
-.THNDRQ100 .MSG ZMA_INFO THNDR DOLIE SRQ Q100
-.THNDRQ110 .MSG ZMA_INFO THNDR SMELZ Q110
-.THNDRQ116 .MSG ZMA_INFO THNDR SMELZ Q116
-.THNDRQ118 .MSG ZMA_INFO THNDR KPASA Q118
-.HEDLYQ77 .MSG ZMA_INFO HEDLY STYMY Q77
-.HEDLYQ87 .MSG ZMA_INFO HEDLY MATLK Q87
-.ARKESQ77 .MSG ZMA_INFO ARKES SEAZY MATLK Q77
-.ARKESQ87 .MSG ZMA_INFO ARKES SEAZY ONEWY Q87
+.MY280 .MSG ZMA_INFO WINCO CHRRI DOLIE Y280
```
Comment (erikquinn) [inline docs/alias/Alias.txt:734]: Please add support for the more verbose MIA:
`.MY280` / `.MIAY280` / `.FLLY280` / `.FY280`

File: docs/alias/Alias.txt [hunk 2]
```diff
@@ -79,6 +79,15 @@ CLEARANCE DELIVERY
 .ABLERTE Please advise if you are able to accept the following route: $1
 .ABLEDEP Please advise if you are able to accept the following departure: $1
 
+----- Flightaware IFR Route Analyzer -----
+.ROUTEM .openurl https://flightaware.com/analysis/route.rvt?origin=KMIA&destination=$uc($1)
+.ROUTEF .openurl https://flightaware.com/analysis/route.rvt?origin=KFLL&destination=$uc($1)
+.ROUTET .openurl https://flightaware.com/analysis/route.rvt?origin=KTPA&destination=$uc($1)
+.ROUTEP .openurl https://flightaware.com/analysis/route.rvt?origin=KPBI&destination=$uc($1)
+.ROUTEE .openurl https://flightaware.com/analysis/route.rvt?origin=KEYW&destination=$uc($1)
+.ROUTER .openurl https://flightaware.com/analysis/route.rvt?origin=KRSW&destination=$uc($1)
```
Comment (erikquinn) [inline docs/alias/Alias.txt:88]: After the below change, why not just remove these as they have no benefit over `.route`?

File: docs/alias/Alias.txt [hunk 3]
```diff
@@ -79,6 +79,15 @@ CLEARANCE DELIVERY
...
+.ROUTE .openurl https://flightaware.com/analysis/route.rvt?origin=$uc($1)&destination=$uc($2)
```
Comment (erikquinn) [inline docs/alias/Alias.txt:89]: Why not simply pull the FP data? eg:
`.ROUTE .openurl https://flightaware.com/analysis/route.rvt?origin=$dep&destination=$arr`

File: docs/alias/Alias.txt [hunk 4]
```diff
@@ -400,6 +409,7 @@ RADAR
 .DMA Descend and maintain $uc($1), the $uc($2) altimeter $altim($2).
 
 .HI $radioname().
+.HIEA $radioname(), expect $uc($1) Runway $uc($2) approach.
```
Comment (erikquinn) [inline docs/alias/Alias.txt:412]: 🎉

**Review by erikquinn — CHANGES_REQUESTED** [review]

File: docs/alias/Alias.txt (hunk 2 above)
Comment (migte) [reply docs/alias/Alias.txt:88]: even more good idea

File: docs/alias/Alias.txt (hunk 1 above)
Comment (migte) [reply docs/alias/Alias.txt:734]: Rgr

**Comment by migte** [comment]
@erikquinn Since metroplex and all the changes that have happened, I can finish up this branch/PR to include the .route commands but I should leave out, if not clean out, all the hedly arkes etc stuff, right?

**Comment by erikquinn** [comment]
@migte Yeah, I guess update what you can, and what you can't, _comment it out_. Leaving it still present and commented out will make it easier in the future when we come back to update and reimplement the same concept with the newer routes and such.

**Comment by erikquinn** [comment]
@migte bump?

**Comment by erikquinn** [comment]
or @ksnifte, maybe you could clean this one up? Want to get it done before #173.

**Comment by ksnifte** [comment]
Is the only thing left to do just clean out the pre-Metroplex DPs?

**Comment by erikquinn** [comment]
yep, to remove any changes made irrelevent by metroplex, and address any outstanding comments. Then request a review from me again. Planning to merge this one first.

File: docs/alias/Alias.txt (hunk 1 above)
Comment (erikquinn) [reply docs/alias/Alias.txt]: no longer relevant, have been removed.

File: docs/alias/Alias.txt (hunk 2 above)
Comment (erikquinn) [reply docs/alias/Alias.txt]: reduced to one

**Pull request merged**